styles.css

```diff
@@ -114,25 +114,22 @@ h1 {
   width: 50px;
 }
 
-.icon-study:active,
-.icon-study:focus {
-  border: 2px solid #B3FD78;
+.icon-study-active {
+  border: 2px solid #B3FD78 !important;
   outline: 0;
-  color: #B3FD78;
+  color: #B3FD78 !important;
 }
 
-.icon-exercise:active,
-.icon-exercise:focus {
-  border: 2px solid #FD8078;
+.icon-exercise-active {
+  border: 2px solid #FD8078 !important;
   outline: 0;
-  color: #FD8078;
+  color: #FD8078 !important;
 }
 
-.icon-meditate:active,
-.icon-meditate:focus {
-  border: 2px solid #C278FD;
+.icon-meditate-active {
+  border: 2px solid #C278FD !important;
   outline: 0;
-  color: #C278FD;
+  color: #C278FD !important;
 }
```

Comment:
also great use of !important. I couldn't figure out how to keep that border, and this is a good solution.